app.js

```diff
@@ -1,4 +1,4 @@
-var Airtable = require("airtable");
+const { Client } = require("pg");
```

Comment:
eeeek. maybe it's good time to start adopting ES syntax.

Reply:
added :)